### ISSUE TITLE
Can ifcopenshell.draw be used on single product types / occurrences?

### ISSUE THREAD

**Issue by Moult** [IfcOpenShell/IfcOpenShell#3675]
One of the features needed for a basic drawing pack are things like window schedules, door schedules, and louvre schedules.

Right now the SVGs are generated via actual camera / cutting planes in the model. If instead a type or occurrence could be selected and only that could be rendered with some settings (e.g. cut midway as a plan, or top down or front or side elevation) that would allow us to generate little HLR thumbnails of different windows, doors, and so on. Something like this:

![2023-08-31-113339_695x491_scrot](https://github.com/IfcOpenShell/IfcOpenShell/assets/88302/bb754f8c-218e-4ffc-b8a2-a58a7d81bc6c)

**Comment by aothms** [comment]
My preferred way of attacking this would still be then to create an IfcAnnotation/DRAWING from the occurrence local placement. If there is no occurrence (only type object) first insert a dummy occurrence. All programmatically of course. what do you think?

**Comment by Moult** [comment]
Indeed, that can therefore all be implemented in Python (perhaps with project.append_asset to add a new occurrence into a fresh model)
